docs/organization/integrations/notification-incidents/index.mdx

```diff
@@ -4,6 +4,7 @@ sidebar_order: 20
 description: "Learn more about Sentry's notification and incidents integrations."
 ---
 
+- [All Quiet](https://allquiet.app/integrations/inbound/sentry/)
```

Comment:
```suggestion
- [All Quiet](https://docs.allquiet.app/integrations/inbound/sentry)
```
Hey there, it makes more sense to link to your docs with instructions on how to set up the integration rather than a marketing page.

Reply:
Yep you're right, commited your suggestion. Thanks